mlflow/environment_variables.py

```diff
@@ -511,3 +511,11 @@ def get(self):
 MLFLOW_GATEWAY_RATE_LIMITS_STORAGE_URI = _EnvironmentVariable(
     "MLFLOW_GATEWAY_RATE_LIMITS_STORAGE_URI", str, None
 )
+
+#: Specifies whether or not to have mlflow configure logging on import.
+#: If set to True, mlflow will configure ``mlflow.<module_name>`` loggers with
+#: logging handlers and formatters.
+#: (default: ``True``)
+MLFLOW_LOGGING_CONFIGURE_LOGGING = _BooleanEnvironmentVariable(
```

Comment:
Can this be renamed to `MLFLOW_CONFIGURE_LOGGING`?

Reply:
Sounds good. I wasn't sure what to call it since there's quite a few references to "logging" things in the repo 😸.